examples/wifi-echo/server/esp32/main/DeviceNetworkProvisioningDelegate.cpp

```diff
@@ -20,7 +20,8 @@
 
 using namespace ::chip;
 
-void ESP32NetworkProvisioningDelegate::ProvisionNetwork(const char * ssid, const char * passwd)
+extern CHIP_ERROR SetWiFiStationProvisioning(const char * ssid, const char * key);
```

Comment:
could we have a header for these?

Reply:
what purpose does 'extern' serve since it is not "C" so linkage should be implied as C++?

Reply:
exporting via header file should fix this.